### ISSUE TITLE
Failed to find tsconfig files being extended with "ENOENT: no such file or directory, ..."

### ISSUE THREAD

**Issue by jeremymeng** [webpro/knip#480]
we have a `tsconfig.json` file that extends another one ("../../../tsconfig.package").  Looks that knip is looking for the file /home/meng/git/jssdk/tsconfig.package but it should have include the ".json" extension

```
meng ➜ ~/git/jssdk/sdk/tables/data-tables (main) $ bunx knip
Analyzing workspace ....
Error loading /home/meng/git/jssdk/tsconfig.package
Reason: ENOENT: no such file or directory, open '/home/meng/git/jssdk/tsconfig.package'
meng ➜ ~/git/jssdk/sdk/tables/data-tables (main) $ l /home/meng/git/jssdk/tsconfig.package.json
/home/meng/git/jssdk/tsconfig.package.json
meng ➜ ~/git/jssdk/sdk/tables/data-tables (main) $ cat tsconfig.json
───────┬────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
       │ File: tsconfig.json
───────┼────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
   1   │ {
   2   │   "extends": "../../../tsconfig.package",
   3   │   "compilerOptions": {
   4   │     "outDir": "./dist-esm",
   5   │     "declarationDir": "./types",
   6   │     "lib": ["dom", "esnext"],
   7   │     "paths": {
   8   │       "@azure/data-tables": ["./src/index"],
   9   │     },
  10   │   },
  11   │   "exclude": ["node_modules", "types", "temp", "browser", "dist", "dist-esm", "./samples/**/*.ts"],
  12   │ }
```

**Comment by webpro** [comment]
Are you using the latest version of Knip?

**Comment by jeremymeng** [comment]
> Are you using the latest version of Knip?

I did `bunx knip@latest`.

**Comment by jeremymeng** [comment]
>bunx knip@latest -V
4.2.1

**Comment by webpro** [comment]
Related: #420 

You'll need to add the `.json` extension for Knip to find it, since this file is not loaded by the TypeScript config loader, but an internal one.

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v4.2.2. See [Release 4.2.2](https://github.com/webpro/knip/releases/tag/4.2.2) for release notes._

**Comment by webpro** [comment]
Oh well I was just cutting a release :)
